task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -90,7 +90,7 @@ spec:
             arch_imageanddigest=$(echo $imagewithouttag@$arch_sha)
 
             echo "Running \"oc image extract\" on image of arch $arch"
-            oc image extract --registry-config ~/.docker/config.json $arch_imageanddigest --path="/:${destination}" --filter-by-os="linux/${arch}"
+            retry oc image extract --registry-config ~/.docker/config.json "$arch_imageanddigest" --path="/:${destination}" --filter-by-os="linux/${arch}"
```

Comment:
What is our policy on maintaining old versions of these tasks?  I know this came up on a recent arch call but don't remember the results

Reply:
I didn't attend the arch call. I think this change don't need migration. But any share about maintaining old version is appreciated.

Reply:
For build tasks, we decided to [deprecate](https://github.com/konflux-ci/build-definitions/pull/2214) all versions but the latest and only maintain one version going forward